### ISSUE TITLE
Not linting HTML, Can't Find Any Errors

### ISSUE THREAD

**Issue by l33tbmb** [AtomLinter/linter-eslint#1129]
## Issue Type
"Question"

## Issue Description
New install of ESLint plus ESLint-Plugin-HTML using Global install (npm install eslint -g) on Windows.
The configuration is linting .js files fine, but will not report results of .html files. Atom UI is listing "Linter Errors" and "Linter Warnings" on open HTML files, and using the Eslint Fix command in Atom generates a success-like message ```Linter-ESLint: Fix attempt complete, but linting errors remain.``` but I cannot view what these Linter messages are. I haven't found anywhere in Atom that is reporting any errors or debug-related output to explain what's happening.

## Bug Checklist
- [x] Restart Atom
Every time I updated the rules, plugin settings, or anything else
- [x] Verify the `eslint` CLI gives the proper result, while `linter-eslint` does not
The CLI is giving the expected results with my rule set:
![image](https://user-images.githubusercontent.com/4237751/38841358-dfd4e150-4198-11e8-9443-968590056b09.png)
- [x] Paste the output of the `Linter Eslint: Debug` command from the Command Palette below

```
Atom version: 1.25.1
linter-eslint version: 8.4.1
ESLint version: 4.19.1
Hours since last Atom restart: 0.1
Platform: win32
Using global ESLint from: C:\Users\USERNAME\AppData\Roaming\npm\node_modules\eslint
Current file's scopes: [
  "text.html.basic",
  "meta.tag.script.html",
  "source.js.embedded.html"
]
linter-eslint configuration: {
  "eslintRulesDirs": [
    "C:\\Projects\\eslinter"
  ],
  "eslintrcPath": "C:\\Projects\\eslinter",
  "globalNodePath": "C:\\Users\\USERNAME\\AppData\\Roaming\\npm",
  "lintHtmlFiles": true,
  "useGlobalEslint": true,
  "showRuleIdInMessage": true,
  "disableWhenNoEslintConfig": true,
  "advancedLocalNodeModules": "",
  "disableEslintIgnore": false,
  "disableFSCache": false,
  "fixOnSave": false,
  "scopes": [
    "source.js",
    "source.jsx",
    "source.js.jsx",
    "source.babel",
    "source.js-semantic"
  ],
  "rulesToSilenceWhileTyping": [],
  "rulesToDisableWhileFixing": [],
  "ignoreFixableRulesWhileTyping": false
}
```

For giggles, here is the rule file that I've put into the root of the project folder
```
{
	"parserOptions": {
		"ecmaVersion": 2017,
		"sourceType": "module"
	},
	"extends": "eslint:recommended",
	"env": {
		"browser": true,
		"node": true
	},
	"rules": {
		"semi": 2,
		"valid-typeof": "warn",
		"import/no-unresolved": "off",
		"import/no-extraneous-dependencies": "off",
		"no-multi-spaces": "off",
		"padded-blocks": "off",
		"linebreak-style": "off",
		"class-methods-use-this": "off",
		"indent": ["error", "tab"],
		"no-unused-vars": ["error", {
			"vars": "local",
			"args": "after-used"
		}],
		"no-multiple-empty-lines": ["error", {
			"max": 2,
			"maxEOF": 1
		}],
		"no-nested-ternary": "off",
		"no-console": ["warn", { "allow": ["warn", "error"] }],
		"object-shorthand": ["error", "methods"]
	},
	"plugins": [
		"html"
	]
}
```

**Comment by skylize** [comment]
You are using a global installation of `eslint`. Did you also install `eslint-plugin-html` globally? Global linting does not use project-level plugins.

**Comment by l33tbmb** [comment]
Yes, both are installed globally. I double checked in the global node_modules folder for both, and the CLI linter was run where neither eslinter modules are installed locally.

**Comment by IanVS** [comment]
Do you happen to be using `atom-ide-ui`? (see https://github.com/AtomLinter/linter-eslint/issues/1071#issuecomment-356998669)

**Comment by l33tbmb** [comment]
I'll have to double check, I'm using whatever was installed by default. Didn't change many settings after install.

**Comment by l33tbmb** [comment]
I took a look through installed packages, in case I threw that in there at some point, and it doesn't look like I'm using ```atom-ide-ui```

I should have mentioned in the report, the HTML files report Linter messages, but it's not the same number or type as the CLI report.

Atom HTML Linter reports 7 warnings, the CLI reports the attached screenshot with 19 Errors and 5 Warnings. I feel like there's some error message tucked away that I can't find that might explain what's actually going wrong.

**Comment by l33tbmb** [comment]
I removed my settings for ```eslintRulesDirs``` and ```eslintrcPath``` then started typing in an HTML file and linting kicked in.
I'm not sure if it was always working but never fired until I started modifying an HTML file, or if removing those settings is what fixed it but everything appears to be working correctly now.

**Comment by skylize** [comment]
Great. Glad you got it worked out.
